docs/new-style-script-objects.md

```diff
@@ -17,8 +17,22 @@ For example,
             const { AppConstants } = ChromeUtils.import(
                 'resource://gre/modules/AppConstants.jsm'
             );
-            return { ...AppConstants };
+            return AppConstants;
         }
     };
 
 defines a new-style script object that requires access to the browser's privileged API in order to fetch its constants.
+
+The equivalent functionality can be accomplished asynchronously:
+
+    module.exports = {
+        requires: { privilege: true },
```

Comment:
Indentation looks large here.

Reply:
Unfortunately it's not -- it's just four spaces per indentation level. It's sort of the default that MD gave me. It's exactly the same as the existing example but looks wider because of the depth.